parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -792,6 +780,31 @@ pub mod pallet {
 				},
 			}
 		}
+
+		pub fn initial_sync(
```

Comment:
Does this need to be `pub` if it is only called from `GenesisBuild::build`?

Reply:
Good point! I changed it to `pub(super)` so that the tests can still access it.